CMakeLists.txt

```diff
@@ -72,6 +72,7 @@ set(VST3_SDK_PATH "" CACHE PATH "Path to VST3_SDK. SDK version >= ${VST3_SDK_VER
 # Modules (alphabetical order please)
 option(MUE_BUILD_APPSHELL_MODULE "Build appshell module" ON)
 option(MUE_BUILD_AUTOBOT_MODULE "Build autobot module" ON)
+option(MUE_BUILD_BRAILLE_MODULE "Build braille module (not yet implemented)" ON)
```

Comment:
please add disabling this module for VTest configuration https://github.com/musescore/MuseScore/blob/master/SetupConfigure.cmake#L109

Reply:
Done!